### PR TITLE
Update FAQ.md

### PR DESCRIPTION
**Pull request by jjasghar**
Updated the correct location for some documents, and removed some `-`.

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -123,21 +123,21 @@ More information about the Granite-7b can be found on the [Hugging Face project
...
-  - [**Freeform compositional skills**](https://github.com/instruct-lab/community/blob/main/docs/SKILLS_GUIDE.md#freeform-compositional-skills) are performative skills that do not require additional context. For example, to train an AI model to write a poem, you would provide examples of poems.
-  - [**Grounded compositional skills**](https://github.com/instruct-lab/community/blob/main/docs/SKILLS_GUIDE.md#grounded-compositional-skills) are performative skills that require additional context. One example is how an AI model reads the value of a cell in a table layout. To create the grounded skill to read a table formatted in Markdown, the additional context might be an example table layout.
+- [**Composition skills.**](https://github.com/instructlab/community/blob/main/docs/SKILLS_GUIDE.md#compositional-skills) Composition or _performative_ skills allow AI models to perform specific tasks or functions. With InstructLab, there are two types of composition skills:
```
Comment (bjhargrave) [inline FAQ.md:130]: I think these linked files are now in the taxonomy repo.

```suggestion
- [**Composition skills.**](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#compositional-skills) Composition or _performative_ skills allow AI models to perform specific tasks or functions. With InstructLab, there are two types of composition skills:
```

File: FAQ.md [hunk 2]
```diff
@@ -123,21 +123,21 @@ More information about the Granite-7b can be found on the [Hugging Face project
...
+  - [**Freeform compositional skills**](https://github.com/instructlab/community/blob/main/docs/SKILLS_GUIDE.md#freeform-compositional-skills) are performative skills that do not require additional context. For example, to train an AI model to write a poem, you would provide examples of poems.
```
Comment (bjhargrave) [inline FAQ.md:131]:
```suggestion
  - [**Freeform compositional skills**](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#freeform-compositional-skills) are performative skills that do not require additional context. For example, to train an AI model to write a poem, you would provide examples of poems.
```

File: FAQ.md [hunk 3]
```diff
@@ -123,21 +123,21 @@ More information about the Granite-7b can be found on the [Hugging Face project
...
+  - [**Grounded compositional skills**](https://github.com/instructlab/community/blob/main/docs/SKILLS_GUIDE.md#grounded-compositional-skills) are performative skills that require additional context. One example is how an AI model reads the value of a cell in a table layout. To create the grounded skill to read a table formatted in Markdown, the additional context might be an example table layout.
```
Comment (bjhargrave) [inline FAQ.md:132]:
```suggestion
  - [**Grounded compositional skills**](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#grounded-compositional-skills) are performative skills that require additional context. One example is how an AI model reads the value of a cell in a table layout. To create the grounded skill to read a table formatted in Markdown, the additional context might be an example table layout.
```

**Review by bjhargrave — CHANGES_REQUESTED** [review]

**Review by lhawthorn — APPROVED** [review]

**Comment by lhawthorn** [comment]
@jjasghar Please rebase so I can do a final review, but what I have reviewed thus far LGTM.

**Comment by bjhargrave** [comment]
@mergifyio rebase

**Review by bjhargrave — APPROVED** [review]

**Review by kelbrown20 — APPROVED** [review]

**Pull request merged**